docs/SQL/development/visual-editor/visual-expression-builder/use-the-visual-expression-builder.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Use the Visual Expression Builder
+id: use-the-expression-builder
+description: Use the Expression Builder
+sidebar_position: 2
+tags:
+  - how-to
+  - development
+  - visual
+  - functions
+  - expression
+  - sql
+---
+
+Develop your SQL expressions by using the Visual Expression Builder, which shows you available columns and functions that you can use to build your expressions.
+
+## Navigate to the Visual Expression Builder
+
+The Visual Expression Builder is supported wherever you see Visual and Code views within your transformation Gems.
+
+- Create or open an existing transformation Gem, and select the **Visual** view.
+
+:::note
+
+The view you select persists across your Gems.
+
+:::
+
+All of the expressions you build using the Visual Expression Builder are converted to code in the Gem and Model Code views. Similarly, edits you make in the Code view, whether it's in an expression or condition editor, will be converted into Visual view.
+
+This allows you to take advantage of both Visual and Code views when building your expressions.
+
+## Build an expression
+
+Using the Visual Expression Builder, you can build an expression using the following modes:
+
+- **Simple Expression**: This mode prompts you to add a target column and an expression.
```

Comment:
Simple Expression gives the examples but Comparision explains the usage. i think this should be consistent across

`Simple expression`: Allows user to choose an expression, like a column, function, Case statement etc;

`Comparision`: Allows user to compare 2 simple expressions

`Grouping`: Allows user to build complex conditions by allowing the combinations of `Comparision` expressions via `AND` or `OR`

Reply:
We use `Grouping` in Filter gem, `Comparision` in IF and ELSE IF in Code Table, everywhere else we use the Simple Expression

Reply:
Statements I wrote are just for your understanding, please re-write those in correct context you are using

Reply:
Done.